.github/CODEOWNERS

```diff
@@ -655,6 +655,11 @@
 /core/domain/draft_upgrade_services*.py @DubeySandeep
 
 
+# Data stability.
+/core/controllers/domain_objects_validator*.py @aks681
+/core/controllers/payload_validator*.py @aks681
```

Comment:
Please assign the files to @rohitkatlaa and add a TODO to add @vojtechjelinek as a codeowner once he returns

Reply:
Done